.changeset/config.json

```
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.1/schema.json",
+  "changelog": [
+    "@changesets/changelog-github",
+    {
+      "repo": "smartcontractkit/chainlink"
```

```suggestion
      "repo": "smartcontractkit/ccip"
```

We probably want @momentmaker's [PR](https://github.com/smartcontractkit/ccip/pull/594) though.